packages/scalars/README.md

```diff
@@ -0,0 +1,78 @@
+# Powerhouse Scalars
+
+This repository contains custom GraphQL scalars used by Powerhouse.
+
+## Adding New Scalars
+
+To add a new scalar to this repository, follow these steps:
+
+1. **Create a New Scalar File**:
+
+    - Navigate to the `src/scalars` directory.
+    - Create a new file for your scalar, e.g., `MyScalar.ts`.
+
+2. **Define the Scalar**:
+
+    - Implement the scalar using the `GraphQLScalarType` from `graphql` library or custom logic.
+    - Ensure you export the scalar properly.
+
+    ```typescript
+    import { GraphQLScalarType, Kind } from 'graphql';
```

Comment:
Let's update this with the required exports after the other comments.
Let's use named exports only. (No export default)